coriolis/wsman.py

```diff
@@ -86,9 +87,17 @@ def from_connection_info(cls, connection_info):
     def disconnect(self):
         self._protocol = None
 
+    def set_timeout(self, timeout):
+        if timeout > 0:
```

Comment:
This bears the risk of of a `TypeError` if `None` somehow gets passed here so please either:
a) account for a timeout of `None` meaning "no timeout at all"
b) add a check for `timeout is not None`, but in that case a simple `if timeout:` would be semantically identical.